lang/python/python3-host.mk

```diff
@@ -8,9 +8,6 @@
 # Note: include this after `include $(TOPDIR)/rules.mk in your package Makefile
 #       if `python3-package.mk` is included, this will already be included
 
-ifneq ($(__python3_host_mk_inc),1)
-__python3_host_mk_inc=1
```

Comment:
i think i did this because i was paranoid at the time, and wasn't too sure about Makefile stuff and potential order of inclusion [like in C];
with `python-host.mk` being included before `python-package.mk` or vice-versa;

this looks fine to remove;